python/dgl/frame.py

```diff
@@ -181,6 +212,9 @@ def to(self, device, **kwargs): # pylint: disable=invalid-name
         Column
             A new column
         """
+        if isinstance(self.storage, distributed.dist_tensor.DistTensor):
+            # copy actual tensor from DistTensor, if self.storage is a DistTensor
+            self._copy_dist_tensor()
```

Comment:
Is this needed? `Column.data` will call `_copy_dist_tensor` anyway.

Reply:
I considered a case where users can call `to()` without accessing attributes. [Column.clone()](https://github.com/dmlc/dgl/blob/master/python/dgl/frame.py#L257) directly copies `self.storage` to a new `Column` but not `self.data`.